src/vcpkg/base/files.cpp

```diff
@@ -293,9 +293,9 @@ namespace
 #if defined(_WIN32)
         const auto ads = std::find(filename, last, ':'); // strip alternate data streams in intra-filename decomposition
         const auto extension = find_extension(filename, ads);
-#else  // ^^^ _WIN32 / !_WIN32 vvv
+#else                                                    // ^^^ _WIN32 / !_WIN32 vvv
```

Comment:
You need to explicitly install Clang 15.0.7 to be consistent with the formatters in CI.